src/shogun/clustering/KMeans.cpp

```diff
@@ -133,7 +133,7 @@ void KMeans::Lloyd_KMeans(SGMatrix<float64_t> centers, int32_t num_centers)
 					}
 					else
 					{
-						/*  mus(:,j)=zeros(dim,1) ; */
+						/*  cluster_centers(:,j)=zeros(dim,1) ; */
```

Comment:
Id suggest to remove this comment as this is evident from the code. Also, one could use a direct zero-ing of the memory here rather than a loop

Reply:
yup, you could  `cluster_centers.get_column_vector(cluster_assignments_i)->zero()`

Reply:
cool, i didn't find a good way to do this zero-ing.